agent/engine/task_manager.go

```diff
@@ -1559,3 +1569,37 @@ func (mtask *managedTask) waitForStopReported() bool {
 	}
 	return taskStopped
 }
+
+func (mtask *managedTask) UnstageVolumes(csiClient csiclient.CSIClient ) error {
```

Comment:
Tests for this new function?

Reply:
Let's at least track the work to add tests somewhere if we don't want to add them right now.